src/Dapi/Types/CategorizedTransaction.cs

```diff
@@ -0,0 +1,36 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Dapi.Types {
+    public class CategorizedTransaction {
+        public float amount { get; }
+        public string date { get; }
+        public TransactionType type { get; }
```

Comment:
Use the `TransactionType` from the `TransactionType.cs` file.

Reply:
This field is being assigned a value of the wrong type, in the constructor below.